src/pages/en/core-concepts/framework-components.md

```diff
@@ -214,7 +214,7 @@ Astro components are always rendered to static HTML, even when they include fram
 
 [Astro components](/en/core-concepts/astro-components/) are HTML-only templating components with no client-side runtime. But, you can use a `<script>` tag in your Astro component template to send JavaScript to the browser that executes in the global scope.
 
-📚 Learn more about [client-side `<scripts>` in Astro components](/en/core-concepts/astro-components/#client-side-scripts)
+📚 Learn more about [client-side `<script>` in Astro components](/en/core-concepts/astro-components/#client-side-scripts)
```

Comment:
```suggestion
📚 Learn more about about using [client-side `<script>` tags in Astro components](/en/core-concepts/astro-components/#client-side-scripts)
```
I think this might be my preference.